src/index.js

```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Kindly fix linters, don't try to skip them, they are here to make your code follow best practices and make you a better programmer